### PR TITLE
No errors closes 8520

### LINKED ISSUE #14: Rule 8520 : Date of Birth is after data collection period (must be on or before the end of the census period)
\<PersonBirthDate\> (N00066) must be on or before \<ReferenceDate\> (N00603) or null

### PR DESCRIPTION
**Pull request by GBurrel**
Draft Code closes #14

### CODE REVIEW

**Review by GBurrel — COMMENTED** [review]
Draft code

**Comment by GBurrel** [comment]
Hi @tab1tha - no it appears to be working as expected. Please review it when you have a moment. Thanks,

**Comment by tab1tha** [comment]
The button at the top allows you to edit the title 
image

**Comment by GBurrel** [comment]
Apologies @tab1tha think I got a bit trigger happy with a couple of screens open...

File: cin_validator/rules/cin2022_23/rule_8520.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    # Create some sample data such that some values pass the validation and some fail.
+
+    test_dobs = pd.to_datetime(
+        [
+            "01/06/2021",  # 0 pass
+            "01/06/2022",  # 1 fail
+            "31/08/2021",  # 2 pass
+            "31/12/2022",  # 3 fail
+            pd.NA,  # 4 ignored
+        ],
```
Comment (tab1tha) [inline cin_validator/rules/cin2022_23/rule_8520.py:64]: Good test coverage.
You've accounted for valid, invalid and missing dates.

**Review by tab1tha — APPROVED** [review]
This is good work @GBurrel !

**Pull request merged**